src/services/envVersion/index.ts

```diff
@@ -29,6 +30,8 @@ const liveGetEnvVersion = async (
   } catch (e) {
     if (e instanceof ValidationError) {
       Sentry.captureException(e);
+    } else if (e.message === "Invalid authentication token provided") {
+      throw new AuthInvalidError(e.message);
     }
     throw new EnvVersionError(e.message);
```

Comment:
hmm actually for this, any idea why `ValidationError` will just throw a `EnvVersionError`?
in this case, actually any type of error (including `ValidationError`) will be thrown as `EnvVersionError`, so maybe can remove the validation error condition, and also can capture all exceptions with sentry, wdyt?

Reply:
will leave it as is!